test/e2e/aws_test.go

```diff
@@ -339,11 +338,11 @@ func TestAWSUpdateCredentials(t *testing.T) {
 	testframework.MustEnsureClusterOperatorStatusIsSet(t, client)
 
 	// Create the image-registry-private-configuration-user secret using the invalid credentials
-	err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+	err = wait.PollImmediate(1*time.Second, testframework.AsyncOperationTimeout, func() (stop bool, err error) {
 		if _, err := util.CreateOrUpdateSecret(imageregistryv1.ImageRegistryPrivateConfigurationUser, imageregistryv1.ImageRegistryOperatorNamespace, fakeAWSCredsData); err != nil {
-			return fmt.Errorf("unable to create secret %q: %#v", fmt.Sprintf("%s/%s", imageregistryv1.ImageRegistryOperatorNamespace, imageregistryv1.ImageRegistryPrivateConfigurationUser), err)
+			return false, nil
```

Comment:
you should log this error so we can see why it was failing when it eventually times out.

Reply:
updated